components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -573,8 +573,19 @@ bool MigrateConfirmationsStateOnFileTaskRunner(
       rewards_service_base_path.AppendASCII("confirmations.json");
 
   if (base::PathExists(legacy_confirmations_state_path)) {
+    const base::FilePath ads_service_base_path = path.AppendASCII("ad_service");
```

Comment:
Should this be "ads_service" or is that part of the fix?

Reply:
:-) good spot as I just spotted that during my final testing. Fixed